### PR TITLE
fix: wrong chart source urls

### PR DESCRIPTION
**Pull request by viceice**
source url for drone helm chart is wrong

### CODE REVIEW

File: charts/drone/values.yaml [hunk 1]
```diff
@@ -159,7 +159,7 @@ env:
   ## REQUIRED: Set the user-visible Drone hostname, sans protocol.
   ## Ref: https://docs.drone.io/installation/reference/drone-server-host/
   ##
-  DRONE_SERVER_HOST: ""
+  DRONE_SERVER_HOST: drone.example.com
```
Comment (jimsheldon) [inline charts/drone/values.yaml:162]: Could this be its own PR, so this PR can focus on the broken URLs? Thanks

**Comment by jimsheldon** [comment]
Thank you for the contribution @viceice. If you can address the comment I made, and resolve the conflicts, I would be happy to merge this.

File: charts/drone/values.yaml (hunk 1 above)
Comment (viceice) [inline charts/drone/values.yaml:162]:
```suggestion
  DRONE_SERVER_HOST: ""
```

**Comment by viceice** [comment]
will fix conflict tomorrow morning

**Comment by viceice** [comment]
@jimsheldon Why `drone-runner-kube` needs a verison bump?

**Comment by viceice** [comment]
@jimsheldon Ready 😉

**Review by jimsheldon — APPROVED** [review]

**Comment by jimsheldon** [comment]
Thanks! merging
